silverback/application.py

```diff
@@ -120,11 +124,14 @@ def broker_task_decorator(
         def add_taskiq_task(handler: Callable) -> AsyncTaskiqDecoratedTask:
             labels = {"task_type": str(task_type)}
 
-            if container and isinstance(container, ContractEvent):
+            # NOTE: Do *not* do `if container` because that does a `len(container)` call,
+            #       which for ContractEvent queries *every single log* ever emitted, and really
+            #       we only want to determine if it is not None
+            if container is not None and isinstance(container, ContractEvent):
                 # Address is almost a certainty if the container is being used as a filter here.
                 if contract_address := getattr(container.contract, "address", None):
                     labels["contract_address"] = contract_address
-                labels["event_signature"] = container.abi.signature
+                labels["event_signature"] = f'"{container.abi.signature}"'
```

Comment:
question: What's up with this?  If it's legit, maybe a comment would clear up any confusion here.

Reply:
I wanted `"Transfer(address indexed sender, ...)"` instead of without quotes, because it might screw up a parser since it contains spaces and commas

Reply:
left a comment

Reply:
What parser?  Should this just be handled in the display code?  I'm leveraging this on the other end by parsing the signature.

Reply:
will handle it in middlewares